### PR TITLE
Get Lubis widht/height from metafile if it's not in db

### PR DESCRIPTION
**Pull request by gjn**
https://github.com/geoadmin/mf-chsdi3/issues/1074

As described in the comments of the issues, we can have a situation where the lubis image is ready (tiled), but the corresponding information (widht and height) is not yet in db. This happens because Lubis image tiles are created constantly, while the db is updated every x month. This resulted in #1074

With this fix, when the widht and height is not yet in the db, it will be taken from the metafile that exists for each lubis image.

### CODE REVIEW

File: chsdi/templates/htmlpopup/lubis.mako [hunk 1]
```diff
@@ -233,7 +271,7 @@ viewer_url = get_viewer_url(request, params)
   </div>
   <br>
 
-% if preview_url != "" and c['attributes']['image_width']:
+% if preview_url != "" and image_width != None:
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/lubis.mako]: can `image_width` still be equal to None at this point?

File: chsdi/templates/htmlpopup/lubis.mako [hunk 2]
```diff
@@ -270,7 +308,7 @@ viewer_url = get_viewer_url(request, params)
       map.highlightFeature('${c['layerBodId']}', '${c['featureId']}');
       map.recenterFeature('${c['layerBodId']}', '${c['featureId']}');
 
-% if preview_url != "" and c['attributes']['image_width']:
+% if preview_url != "" and image_width != None:
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/lubis.mako]: same question here.

**Comment by loicgasser** [comment]
looking good, 2 minor comments. As a side note we should eventually move all the functions in a `.py` file and avoid too much code in mako. (lubis_helpers.py? maybe)

File: chsdi/templates/htmlpopup/lubis.mako (hunk 1 above)
Comment (gjn) [inline chsdi/templates/htmlpopup/lubis.mako]: yes, when the metafile does not exist (meaning lubis tile for this image not yet created, or in process of being created) or some exception thrown in the imagesize_from_metafile function.

Comment (loicgasser) [inline chsdi/templates/htmlpopup/lubis.mako]: ok thanks

**Comment by gjn** [comment]
Thanks for the review!

**Pull request merged**